src/log.ts

```diff
@@ -81,15 +81,15 @@ export function reset() {
   return current;
 }
 
-export function warn(...args: any[]) {
+export function warn(..._: any[]) {
```

Comment:
It's actually not linting any more but enforced by the compiler. It's much better at picking up unused vars as you can see.